nx/lib/nx/serving.ex

```diff
@@ -873,6 +879,11 @@ defmodule Nx.Serving do
       The number of partitions will be determined according to your compiler
       and for which host it is compiling. See the module docs for more information
 
+    * `:weight` - weight used for load balancing when running a distributed serving
```

Comment:
Perhaps

```suggestion
    * `:distribution_weight` - weight used for load balancing when running a distributed serving
```

for the name?

Reply:
Yes, that's better, changed it :+1: